websites/C/Construct 3/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "897580799593938965",
+		"name": "Mur"
+	},
+	"service": "Construct 3",
+	"description": {
+		"en": "Construct 3 is the worlds best game making software. Make your own game in your browser without coding or with Javascript."
+	},
+	"url": "editor.construct.net",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Z9DRjUb.png",
+	"thumbnail": "https://i.imgur.com/Z9DRjUb.png",
```

Comment:
Logo should be 512x512
Thumbnail should be a promotional banner, or a screenshot of the website homepage.

Reply:
> Logo should be 512x512 Thumbnail should be a promotional banner, or a screenshot of the website homepage.

Fixed it.